### ISSUE TITLE
Add Gitlab user OIDC OP 

### ISSUE THREAD

**Issue by EthanHeilman** [openpubkey/openpubkey#199]
Currently we support the gitlab OP for issuing ID Tokens to workflows in gitlab-ci but not the gitlab OP for issuing ID Tokens to users (humans). There is interest in using OpenPubkey in gitlab workspaces: https://gitlab.com/gitlab-org/gitlab/-/issues/438790
This requires support the user auth for the gitlab OP. This should not be difficult to do.

**Comment by SaahilNotSahil** [comment]
@EthanHeilman Is this still required? If yes, then I would like to work on this.

**Comment by EthanHeilman** [comment]
@SaahilNotSahil Yeah, it is a feature that needs to get built. It would be great to have three OpenID Providers on the webchooser.
